instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware/otel_middleware.py

```diff
@@ -341,6 +375,13 @@ def process_response(self, request, response):
             else:
                 activation.__exit__(None, None, None)
 
+        if request_start_time and duration_attrs:
+            duration = max(
+                round((default_timer() - request_start_time) * 1000), 0
+            )
+            self._duration_histogram.record(duration, duration_attrs)
+        if active_requests_count_attrs:
+            self._active_request_counter.add(-1, active_requests_count_attrs)
```

Comment:
Attributes are optional

```suggestion
        if request_start_time is not None:
            duration = max(
                round((default_timer() - request_start_time) * 1000), 0
            )
            self._duration_histogram.record(duration, duration_attrs)
        self._active_request_counter.add(-1, active_requests_count_attrs)
```

Reply:
Done